endpoints.json

```diff
@@ -119,5 +119,23 @@
         "created_at": "2024-05-30T12:50:15.747Z"
       }
     }
+  },
+  "PATCH /api/articles/:article_id": {
+    "description": "patches an article object for an article_id",
+    "queries": [],
+    "exampleResponse": {
+      "article": [
+        {
+          "article_id": 3,
+          "title": "Eight pug gifs that remind me of mitch",
+          "topic": "mitch",
+          "author": "icellusedkars",
+          "body": "some gifs",
+          "created_at": "2020-11-03T09:12:00.000Z",
+          "votes": 100,
+          "article_img_url": "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+        }
+      ]
```

Comment:
This should be the article object directly not an array with a single article object